backends/LxcBackend.py

```diff
@@ -283,15 +288,16 @@ def attach(self, user, run_command):
         lxccontainer.attach_wait(lxc.attach_run_command, run_command, env_policy=lxc.LXC_ATTACH_CLEAR_ENV)
 
 
-class LxcMaster(LxcBackend,Master):
+class LxcMaster(LxcBackend, Master):
     """
     This class defines methods to manage LXC masters
     """
     def __repr__(self):
-        return("{Master " + self.backend + ":" + self.prefix + self.name
-                + "}")
+        return(
```

Comment:
does not fit on one line ? I think it does not since you propose that, but just to be sure (I did not count...)

Reply:
It seems to fit well in one line.